docs/src/examples2.md

```diff
@@ -0,0 +1 @@
+# Examples
```

Comment:
shouldn't this file be called just example.md

Reply:
This was because the examples weren't working. So examples2 was just the empty file until the examples work.